From 11e09019d714f522075e3c3ad66eeeebb3e85d5c Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Wed, 11 Dec 2024 21:01:15 +0100
Subject: [PATCH] media-tuner-si2157: add missing dependency

Add missing dependency on kernel module media-controller for tuner si2157
---
 package/kernel/linux/modules/dvb.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/kernel/linux/modules/dvb.mk b/package/kernel/linux/modules/dvb.mk
index 061f912b6b..f010da8375 100644
--- a/package/kernel/linux/modules/dvb.mk
+++ b/package/kernel/linux/modules/dvb.mk
@@ -882,7 +882,7 @@ $(eval $(call KernelPackage,media-tuner-it913x))
 define KernelPackage/media-tuner-si2157
   TITLE:=Silicon Labs Si2157 silicon tuner
   $(call MediaTuner,si2157,CONFIG_MEDIA_TUNER_SI2157)
-  DEPENDS+=+kmod-regmap-i2c
+  DEPENDS+=+kmod-regmap-i2c +kmod-media-controller
 endef
 
 define KernelPackage/media-tuner-si2157/description
-- 
2.47.1

